### PR TITLE
Dark mode improvements

### PR DESCRIPTION
**Pull request by yoonieaj**
## Proposed Changes

This PR makes improvements to the dark mode version of the website, and adds new versions of the logos. 

...

<details>
<summary>Screenshots of your changes (if applicable)</summary>

Current dark mode:

image

New dark mode:

image

image

</details>

## Type of Change

_(Write an `X` or a brief description next to the type or types that best describe your changes.)_

| Type                                                                                    | Applies? |
| --------------------------------------------------------------------------------------- | -------- |
| 🚨 _Breaking change_ (fix or feature that would cause existing functionality to change) |          |
| ✨ _New feature_ (non-breaking change that adds functionality)                          |          |
| 🐛 _Bug fix_ (non-breaking change that fixes an issue)                                  |          |
| 🎨 _User interface change_ (change to user interface; provide screenshots)              |    x     |
| ♻️ _Refactoring_ (internal change to codebase, without changing functionality)          |          |
| 🚦 _Test update_ (change that _only_ adds or modifies tests)                            |          |
| 📚 _Documentation update_ (change that _only_ updates documentation)                    |          |
| 📦 _Dependency update_ (change that updates a dependency)                               |          |
| 🔧 _Internal_ (change that _only_ affects developers or continuous integration)         |          |

## Checklist

_(Complete each of the following items for your pull request. Indicate that you have completed an item by changing the `[ ]` into a `[x]` in the raw text, or by clicking on the checkbox in the rendered description on GitHub.)_

Before opening your pull request:

-   [x] I have performed a self-review of my changes.
    -   Check that all changed files included in this pull request are intentional changes.
    -   Check that all changes are relevant to the purpose of this pull request, as described above.
-   [ ] I have added tests for my changes, if applicable.
    -   This is **required** for all bug fixes and new features.
-   [ ] I have updated the project documentation, if applicable.
    -   This is **required** for new features.
-   [ ] If this is my first contribution, I have added myself to the list of contributors.
-   [x] I have updated the project Changelog (this is required for all changes).

After opening your pull request:

-   [x] I have verified that the CI tests have passed.
-   [x] I have reviewed the test coverage changes reported by Coveralls.
-   [x] I have [requested a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a project maintainer.

## Questions and Comments

_(Include any questions or comments you have regarding your changes.)_

### CODE REVIEW

**Review by david-yz-liu — COMMENTED** [review]
@yoonieaj the images are looking good; in this PR please also go ahead an incorporate these logos directly into the demo website (you'll need to figure out how to conditionally include a different logo image for different themes)

**Comment by yoonieaj** [comment]
The MUI documentation suggests creating a single theme with a new `colorSchemes` attribute to define different palettes for light and dark mode. For some reason when I tried to do this it wouldn't apply the themes, so I ended up creating two separate themes (the old way of doing it).

File: demo/src/index.tsx [hunk 1]
```diff
@@ -4,22 +4,53 @@ import App from "./App";
...
         },
     },
 });
 
+const darkTheme = createTheme({
+    palette: {
+        mode: "dark",
+        primary: {
+            main: "#89c48c",
+            paper: "#dadada",
```
[the remark this answers is not in the document]
Comment (yoonieaj) [reply demo/src/index.tsx]: I just changed the colour to `#cacaca`! I'm worried there might not be enough contrast with the orange `id` values, can you take a look?

image

**Review by david-yz-liu — APPROVED** [review]
Thanks @yoonieaj! I think the dark mode background colour `#cacaca` looks okay right now, but we can always change it later.

**Pull request merged**